### PR TITLE
Update strengthify to 0.4.2

### LINKED ISSUE #20454: "Very weak password" tooltip does not dissapear in personal settings page
### Steps to reproduce

1. Go to personal settings
2. Try to change your pwd with another very weak and write a wrong pwd in current password

### Expected behaviour

The "very week password" tooltip should dissapear

### Actual behaviour

The tooltip stills there after browsing away or scrolling

Note: The text file is created and it is visible after refreshing the screen.

### Server configuration
**Operating system**: ubuntu 14.04

**Web server:** apache 2.4.7

**Database:** mysql

**PHP version:** 5.5.9

**ownCloud version: ownCloud Enterprise Edition 8.2.1. (RC2)

**Updated from an older ownCloud or fresh install:** Fresh install

**Are you using external storage, if yes which one:** no;
**Are you using encryption:** no
**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Firefox
tooltipw

### PR DESCRIPTION
**Pull request by MorrisJobke**
- includes local changes (ported them to upstream - see #10625)
- tipsy -> tooltip
- re-renders the element on scroll of a parent - fixes #20454 

cc @PVince81 @davitol 

Mostly moved code.

### CODE REVIEW

File: core/vendor/strengthify/jquery.strengthify.js [hunk 1]
```diff
@@ -125,7 +111,24 @@
...
+
+		// add elements
+		$('.strengthify-wrapper')
+			.append('<div class="strengthify-bg" />')
+			.append('<div class="strengthify-container" />')
+			.append('<div class="strengthify-separator" style="left: 25%" />')
+			.append('<div class="strengthify-separator" style="left: 50%" />')
+			.append('<div class="strengthify-separator" style="left: 75%" />');
+
+		me.parents().on('scroll', drawStrengthify);
```
Comment (MorrisJobke) [inline core/vendor/strengthify/jquery.strengthify.js]: This is the actual change.

**Comment by davitol** [comment]
@MorrisJobke The tooltip behaviour improves. Now it doesn't move following the scroll or clicking the options, but this pwd tooltip is still visible after trying to change the pwd, and the current pwd written is wrong. But IMO :+1:

**Comment by PVince81** [comment]
In IE8 it shows the tooltip even for the placeholder initially:
ie8-strengthify

On master it does show the placeholder as dots, but no tooltip appears.
The fact that it shows dots in the first place is another separate bug.

**Comment by PVince81** [comment]
@MorrisJobke it might be easier to "fix" this by making the fields empty when IE8 is detected instead of placeholder.

**Comment by MorrisJobke** [comment]
> @MorrisJobke it might be easier to "fix" this by making the fields empty when IE8 is detected instead of placeholder.

But this is something different to fix.

**Comment by PVince81** [comment]
Ok fine :+1:

**Comment by MorrisJobke** [comment]
@PVince81 can you open tickets for the other stuff you noticed?

**Comment by PVince81** [comment]
Seems it already exists https://github.com/owncloud/core/issues/18820

**Pull request merged**